tests/utils/esx/esxcli.go

```diff
@@ -77,3 +78,39 @@ func WaitForExpectedState(fn getVMPowerStatus, vmName, expectedState string) boo
 	log.Printf("Timed out to poll status\n")
 	return false
 }
+
+// IsVMBackOnline checks if VM is fully back online after being restarted
+func IsVMBackOnline(vmIP, vmName string) bool {
```

Comment:
minor: `IsVMBackOnline` is good but not ideal when `isVDVSRunning=false` .. in that case vm comes back online and vDVS did not. The name does not serve the whole purpose.